README.md

```diff
@@ -0,0 +1,46 @@
+Read More: A Plugin for Pelican
+===============================
+
+[![Build Status](https://img.shields.io/github/workflow/status/pelican-plugins/read-more/build)](https://github.com/pelican-plugins/read-more/actions)
+[![PyPI Version](https://img.shields.io/pypi/v/pelican-read-more)](https://pypi.org/project/pelican-read-more/)
+![License](https://img.shields.io/pypi/l/pelican-read-more?color=blue)
+
+This Pelican plugin inserts an inline “Read More” link into the last HTML element of the summary.
+
+For more information regarding why it was created, please visit: https://www.vuongnguyen.com/read-more-python-lxml/
+
+Installation
+------------
+
+This plugin can be installed via:
+
+    python -m pip install pelican-read-more
+
+Settings
+--------
+
+Following are some settings related to this plugin:
+
+    # Create a summary with the specified length
+    SUMMARY_MAX_LENGTH = 50
+
+    # Define what goes inside the “Read More” link
+    READ_MORE_LINK = None  # Example: "<span>continue</span>"
+
+    # Define the format of the “Read More” link
+    READ_MORE_LINK_FORMAT = '<a class="read-more" href="/{url}">{text}</a>'
+
```

Comment:
Should we add a Contributors Section similar to the PDF Plugin? like:

```
Contributors
------------

Contributors include: Vuong Nguyen (http://vuongnguyen.com), Dashie, Justin Mayer, Kernc
```

Reply:
Sure. I added this section.